client/verta/verta/_cli/registry/update.py

```diff
@@ -106,6 +106,14 @@ def update_model_version(model_name, version_name, label, model, custom_module,
 
 
 def add_attributes(model_version, attribute, overwrite):
-    _multiple_arguments_for_each(attribute, "attribute",
-                                 lambda key, value: model_version.add_attribute(key, json.loads(value), overwrite=overwrite),
+    def action(key, value):
```

Comment:
We should definitely have a more descriptive name for this than "action"!

Reply:
"action" is the name of the parameter in the method =) Maybe call it "logger" or sth?

Reply:
I'm aware that the parameter is named `action` because Evgenii wrote it to accept more general operations, but I don't think that means the function we define here also has to be named `action`, unless I'm missing some quirk in Python.

Reply:
Fixed.